<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>884</width>
                <height>665</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: #ADD8E6</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>140</x>
                        <y>20</y>
                        <width>621</width>
                        <height>91</height>
                    </rect>
                </property>
                <property name="minimumSize">
                    <size>
                        <width>8</width>
                        <height>0</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>Chalkboard SE</family>
                        <pointsize>60</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true">color: #1E2F3C;</string>
                </property>
                <property name="text">
                    <string>NÃºmero de jugadores</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btn3">
                <property name="geometry">
                    <rect>
                        <x>210</x>
                        <y>360</y>
                        <width>141</width>
                        <height>131</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton {
                        background-color: #1d5a96;
                        color: black;
                        font-size: 24px;
                        border-radius: 40px;
                        padding: 20px;
                        }
                    </string>
                </property>
                <property name="text">
                    <string>3</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btn2">
                <property name="geometry">
                    <rect>
                        <x>360</x>
                        <y>180</y>
                        <width>141</width>
                        <height>131</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton {
                        background-color: #1d5a96;
                        color: black;
                        font-size: 24px;
                        border-radius: 40px;
                        padding: 20px;
                        }
                    </string>
                </property>
                <property name="text">
                    <string>2</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btn4">
                <property name="geometry">
                    <rect>
                        <x>510</x>
                        <y>360</y>
                        <width>141</width>
                        <height>131</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton {
                        background-color: #1d5a96;
                        color: black;
                        font-size: 24px;
                        border-radius: 40px;
                        padding: 20px;
                        }
                    </string>
                </property>
                <property name="text">
                    <string>4</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>884</width>
                    <height>24</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
